Assets/MixedRealityToolkit-SDK/Features/Input/Scripts/InputSystemGlobalListener.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem;
+using Microsoft.MixedReality.Toolkit.Internal.Managers;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.Input
+{
+    /// <summary>
+    /// This component ensures that all input events are forwarded to this <see cref="GameObject"/> when focus or gaze is not required.
+    /// </summary>
+    public class InputSystemGlobalListener : MonoBehaviour
+    {
+        private IMixedRealityInputSystem inputSystem = null;
+        protected IMixedRealityInputSystem InputSystem => inputSystem ?? (inputSystem = MixedRealityManager.Instance.GetManager<IMixedRealityInputSystem>());
+
+        protected virtual void OnEnable()
```

Comment:
doc comments

Reply:
was reading quickly.... sorry

Reply:
@davidkline-ms We don't (and likely never will) document the Unity MonoBehaviour events.
Only standard we would impose was to ensure Unity methods are wrapped in a Region called

```#region MonoBehaviour Implementation```

As we have done in other areas of the codebase.